<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computing topological data</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonObjectFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Object file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelObjectFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCompute">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>871</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSkelFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Skeleton file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSkelFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVoxelSize_y">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVoxelSize_y">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel size y (um)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDiam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Fixed diameter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDiam">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDiam">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Diameter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCalib">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCalib">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>161</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibration parameter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVoxelSize_z">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVoxelSize_z">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel size z (um)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVoxelSize_x">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVoxelSize_x">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voxel size x (um)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxUniform">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Uniform vessel diameters?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_junction">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_junctionMax">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Use junction max diameter</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_junctionAve">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Use junction ave diameter</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditMaxRatio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_maxratio">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold ratio max/min diameter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNprunecycles">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_nprunecycles">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of extra prune cycles</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLenlimit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_lenlimit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Length limit for distribution (um)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCompute</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>topology()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonObjectFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>objectFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSkelFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>skelFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDiam</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>diamCheckBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_junctionAve</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>junctionRadioButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>objectFileSelecter()</slot>
  <slot>skelFileSelecter()</slot>
  <slot>outputFileSelecter()</slot>
  <slot>topology()</slot>
  <slot>diamCheckBox()</slot>
  <slot>junctionRadioButton()</slot>
 </slots>
</ui>
